### DISCUSSION TITLE
PKGi PS3 v1.2.4

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/pkgi-ps3#120] [General]
### Added

* Explicit FTP over TLS support
* Increased database memory limit to 32 Mb
* Variable Menu width (auto-adjusted to translated text width)

### Fixed

* Progress bar not showing correct values when resuming a file
* Proper screen aspect ratio (Tiny3D issue)

<hr /><em>This discussion was created from the release <a href='https://github.com/bucanero/pkgi-ps3/releases/tag/v1.2.4'>PKGi PS3 v1.2.4</a>.</em>

**Comment by caloy12** [comment]
Hello sir can u add farcry 4 on pkgi 😔

**Reply by Masamune3210** [reply]
Pkgj does not supply the games, it just allows you to download them.

**Reply by SamuelyhGAEL2026** [reply]
A loja está dando erro na hora de atualizar baixei ela aí mas está dando erro

**Comment by Fredinafazbearuvu** [comment]
the store does not add the dlcs to my game and I already installed them several times

**Reply by bucanero** [reply]
> the store does not add the dlcs to my game and I already installed them several times

The application doesn't provide any link or content. You must use your own files.

**Comment by BenT1no** [comment]
is there a way to download .rap files on pkgi if yes tell me if no please add

**Reply by ZeRo002YT** [reply]
you can just use psn patch no ban no bullsh/t just open it and press L1 wait few sec and press start and you good to go 

-RAP File  MUST BE IN SMALLCASE .rap

- the .rap must be in a fat32 type of hard/usb and must  creat exdata to put the .rap in it so psn patch reads it
- play lol

**Reply by Masamune3210** [reply]
Please don't tell people there is no chance of being banned with the patches. There is always a chance, the patches are not prescient, they are reactionary.

**Comment by sniperassault** [comment]
Hola Damian! Gracias por hacer esta genial aplicacion. Queria preguntarte si hay alguna manera de poder descargar archivos de mediafire poniendolos en la base de datos, ya que lo he intentado pero me tira un error al momento de descargar. Seguramente sea porque para poder descargar desde mediafire tenes que cliquear en el boton que dice "Download", cosa que para los juegos de psn no es nesesario porque tienen descarga directa pero talvez vos sepas alguna manera de poder descargar de igual forma. Muchas gracias!!

**Reply by bucanero** [reply]
No, esos sitios de descarga en general no funcionan ya que no ofrecen links directos y requieren interaccion del usuario. La opción es descargar en PC y luego copiar el archivo, o ejecutar un servidor web local con el archivo y usar ese link local.

**Reply by sniperassault** [reply]
Me lo imaginaba... Gracias por aclararme la duda! Saludos!!

**Comment by jip21345** [comment]
Its showing black screen when i try load it

**Comment by Jow25** [comment]
Please add all skyalnders games in the PKGI!!!!

**Reply by Masamune3210** [reply]
Reading is hard for you, huh. For the third time on this very page, PKGj does not supply the links to the packages, it just makes it easier for you to download them.

**Reply by ebube124567** [reply]
Some guy don't understand

**Comment by Ponguetu** [comment]
Buenas noches la última versión pkgi-ps3 no se ejecuta con cfw, que puedo hacer? Gracias

**Reply by X3r0-V01d** [reply]
It was working for me until I wiped and upgraded my internals. Put it back on, and it gives me the error when I open it up, but doesn't let me refresh, change search specifications, or anything

**Comment by Brx6600xt** [comment]


**Reply by bucanero** [reply]
It's probably a package that is in XML format, those are not supported. See more info here: https://github.com/bucanero/pkgi-ps3/issues/34

**Comment by TheCrow1137** [comment]
When I try to open PKGi 1.2.4  with HFW 4.91 and HEN 3.3.0 , the screen goes dark.

**Comment by Glayciel** [comment]
Well, sorry for the somewhat disjointed comment, but would anyone, not necessarily the Devs of the app, have or could provide a safe and reliable method or link to download games like Skyrim, Fallout New Vegas and Dark souls 1? I've never used Github, or at least I've never been very involved, so sorry for the nonsensical question. Just asking because I've been looking for these games for ages and I don't have a PC or know of any reliable method of getting these games to transfer to the console.

**Reply by Masamune3210** [reply]
The......app that this page is for?

**Reply by Glayciel** [reply]
Well..yes? Why?

**Reply by Masamune3210** [reply]
How should I know, you are the one that asked lol. PKGj's entire point in existing is to allow for games to be installed to the console from the console itself.

**Comment by ebube124567** [comment]
I'm trying to download it but it's not downloading

**Comment by CyPha-SaRin** [comment]
Awesome app, would be handy to have a show installed only option to narrow the list down like that when needed. Found it would be helpful when going back thru the list to scan for updates, maybe a scan all installed for updates option may be better? Would also like to see an Australian region option if enough games were released here deeming that worth it. And maybe even a restart ps3 button.
A bonus would be an original platform filter, PS1 games, PS2, etc, but sorting by Title also kinda does this so not the worst without it.

**Reply by Masamune3210** [reply]
App is limited to the region delineation that the system itself uses, it has no way of knowing anything other than what the titleid provides

**Comment by phoenix200mac-ui** [comment]
im so helplessly lost, The first download is working but i cant get any of the txt stuff working. What do i do with the zip file and zerg file, I noticed the pkg download is called dev_hdd0... but how do I add a txt file into it and what do i add into the file. Im not sure if im even helpable but if someone knows how to explain it'd be so appreciated. i have the evil firmware not HEN.

**Reply by Masamune3210** [reply]
What are you talking about? All PKGi is for is installing the game. It doesn't do anything with the files once it's installed, so I don't know where you got txt files, zip files and whatever the fsck a zerg file is.

**Comment by phoenix200mac-ui** [comment]
It was my first time downloading anything really I started downloading
developer files or something random zip files from your guys website. Got
it up n running after 6 hours of doing the wrong thing. Thank you guys
